asserts/device_asserts.go

```diff
@@ -45,6 +45,16 @@ func (mod *Model) Model() string {
 	return mod.HeaderString("model")
 }
 
+// DisplayName returns the display-friendly name of the model or
```

Comment:
Small nit, but "display-friendly" looks a bit like a header name in this context. Maybe "… returns the friendly `display-name` of the model …" instead?